topi/python/topi/x86/conv2d.py

```diff
@@ -40,6 +40,18 @@ def _create_tuning_space(cfg, data, kernel, strides, padding, dilation, layout):
     if layout == 'NCHW':
         n, ic, h, w = dshape
         oc, _, kh, kw = kshape
+    elif 'NCHW' in layout and 'c' in layout:
+        n, ic_chunk, h, w, ic_bn = dshape
```

Comment:
Shouldn't this do a regex match against `"NCHW(\d+)c"`? This seems unnecessarily fragile (though inline with existing style), but will fail for e.g. NCHW4h4w8c which is something I've been experimenting with.

Reply:
I'm not very familiar with regex, but this doesn't seem to match "NCHW4h4w8c" unless I change the expression to something like `NCHW.+(\d+)c`?